serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces.
+---
+
+This content applies to: <DocBadge template="elasticsearch" slug="/serverless/elasticsearch/what-is-elasticsearch-serverless" /> <DocBadge template="security" slug="/serverless/security/what-is-security-serverless" />
```

Comment:
It's a bit more nuanced. All project types have Spaces available, but only ES3 and Security project types will have access to the functionality described by the `Customize access to space` section below.

Reply:
Oh ok, so basically the Space feature is in all, but not custom roles?

Reply:
As it is right now in production:
- Search has both spaces and custom roles
- Security has both spaces and custom roles
- Observability has neither of them

So if I understand you correctly, Spaces but not custom roles will become available in Observability projects shortly? Just trying to make sense of what I can see now vs. the GA plan 😅

Reply:
Currently in production-canary:

- Search has both spaces and custom roles
- Security has both spaces and custom roles
- Observability has spaces, but not custom roles

We plan to enable the same as above in production non-canary on Wednesday, in preparation for GA.

Reply:
Awesome thank you for confirming! I'll make the corresponding changes in this PR to the Space and Custom roles content.